src/routes/auth.nim

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: AGPL-3.0-only
+
+import jester
+
+import router_utils
+import ".."/[types, auth]
+
+proc createAuthRouter*(cfg: Config) =
+  router auth:
+    get "/.well-known/nitter-auth-sha256":
```

Comment:
Can we call this something simpler like account-pool-auth?

Reply:
i renamed it to nitter-auth. `nitter` because there are some standardized paths under .well-known, and not specific to account-pool since the status page at status.d420.de uses the same auth now